<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21602" started-at="2021-06-22T15:48:31Z" finished-at="2021-06-22T15:48:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21602" started-at="2021-06-22T15:48:31Z" finished-at="2021-06-22T15:48:53Z">
      <class name="com.ibanking.testCases.NewCustomerPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="8399" started-at="2021-06-22T15:48:32Z" finished-at="2021-06-22T15:48:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="createCustomerTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="createCustomerTest" duration-ms="2151" started-at="2021-06-22T15:48:40Z" data-provider="newCustData" finished-at="2021-06-22T15:48:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vnrnvueu bnght]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[female]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[22061998]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[sif oesifh oi]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Osfhksg]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Wsdfimm]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[909876]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[9091901128]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[drglkj113350@mail.com]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[1q2w3e4r]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[42631]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.ibanking.testCases.NewCustomerPageTest.createCustomerTest(NewCustomerPageTest.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCustomerTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="792" started-at="2021-06-22T15:48:42Z" finished-at="2021-06-22T15:48:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="setUp" is-config="true" duration-ms="6976" started-at="2021-06-22T15:48:43Z" finished-at="2021-06-22T15:48:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="createCustomerTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="createCustomerTest" duration-ms="1694" started-at="2021-06-22T15:48:50Z" data-provider="newCustData" finished-at="2021-06-22T15:48:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qwmnkl]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[female]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[15122000]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[kj ajf ih99 kjwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Jlkk]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Hhhsiuf]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[887654]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[8790987656]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[fkjlsf876784@mail.com]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[1q2w3e4r]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[empty]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.ibanking.testCases.NewCustomerPageTest.createCustomerTest(NewCustomerPageTest.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCustomerTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ibanking.testCases.NewCustomerPageTest@204f30ec]" name="tearDown" is-config="true" duration-ms="782" started-at="2021-06-22T15:48:52Z" finished-at="2021-06-22T15:48:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ibanking.testCases.NewCustomerPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
